scripts/bootstrap.js

```diff
@@ -95,9 +103,9 @@ function run() {
       command: () => {
         log.info(prefix, 'prepare');
         spawn(
-          `nx run-many --target=prepare --all --parallel ${
+          `nx run-many --target="prepare" --all --parallel ${
             process.env.CI ? `--max-parallel=${maxConcurrentTasks}` : ''
-          }`
+          } -- --optimized`
```

Comment:
I needed this in my prebundle work. It does nothing for now.. I guess i can remove it from this PR.

I struggled quite a bit to find a way to pass flags to the "target" from `nx`. There was no documentation on how to do this, but this works great 🎈